packages/manager/tools/sao-ovh-manager-module/template/src/dashboard/template.html

```diff
@@ -0,0 +1,12 @@
+<div>
+    <oui-header heading="{{:: $ctrl.serviceName }}">
+        <oui-header-tabs>
+            <oui-header-tabs-item
+                href="{{:: $ctrl.dashboardLink }}"
+                active="$ctrl.currentActiveLink() === $ctrl.dashboardLink"
+                >Home</oui-header-tabs-item
```

Comment:
Any reason to not rely on a given translation?

Reply:
The issue is the translation key depends on the module name which is dynamic, prettier currently doesn't tolerate variable declaration in a JSON file and it will be complicated to handle translation request as JSON will not be valid. We could think of generating the files like it is done for package.json but we will have to loop on language list. I don't know what's best

Reply:
I think we can stick with static translations here.